MAVProxy/tools/MAVExplorer.py

```diff
@@ -1176,6 +1183,38 @@ def cmd_paramchange(args):
         vmap[pname] = pvalue
     mestate.mlog.rewind()
 
+def cmd_logmessage(args):
+    '''show log message information'''
+    mlog = mestate.mlog
+    usage = "Usage: logmessage <help|download>"
+    if len(args) > 0:
+        if args[0] == 'help':
+            if len(args) < 2:
+                print(usage)
+                return
+            if hasattr(mlog,'metadata'):
+                mlog.metadata.print_help(args[1])
```

Comment:
```suggestion
            if hasattr(mlog,'metadata'):
                mlog.metadata.print_help(args[1])
            else:
                print("Incompatible pymavlink; upgrade pymavlink?")
```

Reply:
Definately good idea!... I think I will need to also check if its a DF or Telem log, and do a different message, e.g. "Log message help is not supported for Telemetry logs".

Reply:
updated including additoinal check for `isinstance(mlog, mavutil.mavlogfile)`